src/components/ProductVersionPages/ProductVersionPages.tsx

```diff
@@ -0,0 +1,60 @@
+import { useEffect } from 'react';
+import { Outlet, useOutletContext, useParams } from 'react-router-dom';
+
+import { PageTitles } from 'common/constants';
+
+import { IServiceContainer, useServiceContainer } from 'hooks/useServiceContainer';
+import { useTitle } from 'hooks/useTitle';
+
+import { PageLayout } from 'components/PageLayout/PageLayout';
+import { ServiceContainerLoading } from 'components/ServiceContainers/ServiceContainerLoading';
+import { Tabs } from 'components/Tabs/Tabs';
+import { TabsItem } from 'components/Tabs/TabsItem';
+
+import * as productVersionApi from 'services/productVersionApi';
+
+import { generatePageTitle } from 'utils/titleHelper';
+
+type ContextType = { serviceContainerVersion: IServiceContainer };
+
+export const ProductVersionPages = () => {
+  const { productVersionId } = useParams();
+
+  const serviceContainerVersion = useServiceContainer(productVersionApi.getProductVersion);
+  const serviceContainerVersionRunner = serviceContainerVersion.run;
+
+  useEffect(() => {
+    serviceContainerVersionRunner({ serviceData: { id: productVersionId } });
+  }, [serviceContainerVersionRunner, productVersionId]);
+
+  useTitle(
+    generatePageTitle({
+      serviceContainer: serviceContainerVersion,
+      firstLevelEntity: 'Product',
+      nestedEntity: 'Version',
+      entityName: `${serviceContainerVersion.data?.version} ${PageTitles.delimiterSymbol} ${serviceContainerVersion.data?.product.name}`,
```

Comment:
`v` suffix is missing before product version, but let's go without it for now, I will update also our documentation